kiota_abstractions/request_information.py

```diff
@@ -145,8 +153,8 @@ def remove_request_options(self, options: List[RequestOption]) -> None:
 
     def set_content_from_parsable(
         self,
-        request_adapter: Optional["RequestAdapter"],
-        content_type: Optional[str],
+        request_adapter: RequestAdapter,
```

Comment:
wouldn't these changes be source breaking?

Reply:
Not really a breaking change from an end user perspective as the type annotations in Python are not enforced at runtime, they are merely hints. However, we'll need to update the signature in the concrete classes for consistency.